getting-started/ts-example/README.md

```diff
@@ -0,0 +1,354 @@
+
+```typescript
+import { Meter } from "@opentelemetry/metrics";
+import { Metric, BoundCounter } from "@opentelemetry/types";
+
+const meter = new Meter();
+
+const requestCount: Metric<BoundCounter> = meter.createCounter("requests", {
+  monotonic: true,
+  labelKeys: ["route"],
+  description: "Count all incoming requests"
+});
+
+const handles = new Map();
+
+export const countAllRequests = () => {
+  return (req, res, next) => {
+    if (!handles.has(req.path)) {
+      const labelSet = meter.labels({ route: req.path });
+      const handle = requestCount.bind(labelSet);
+      handles.set(req.path, handle);
+    }
+
+    handles.get(req.path).add(1);
+    next();
+  };
+};
+```
+
+Now let's import and use this middleware in our application code:
+
+```typescript
+import { countAllRequests } from "./monitoring";
+const app = express();
+app.use(countAllRequests());
+```
+
+Now, when we make requests to our service our meter will count all requests.
+
+**Note**: Creating a new `labelSet` and `handle` on every request is not ideal as creating the `labelSet` can often be an expensive operation. This is why handles are created and stored in a `Map` according to the route key.
+
+#### Initialize and register a metrics exporter
+
+([link to JavaScipt version](../README.md#initialize-and-register-a-metrics-exporter))
+
+Counting metrics is only useful if we can export them somewhere that we can see them. For this, we're going to use prometheus. Creating and registering a metrics exporter is much like the tracing exporter above. First we will need to install the prometheus exporter.
+
+```sh
+$ npm install @opentelemetry/exporter-prometheus
+```
+
+Next, modify your `monitoring.ts` file to look like this:
+
+```typescript
+import { Meter } from "@opentelemetry/metrics";
+import { Metric, BoundCounter } from "@opentelemetry/types";
+import { PrometheusExporter } from "@opentelemetry/exporter-prometheus";
+
+const meter = new Meter();
+
+meter.addExporter(
+  new PrometheusExporter({ startServer: true }, () => {
+    console.log("prometheus scrape endpoint: http://localhost:9464/metrics");
+  })
+);
+
+const requestCount: Metric<BoundCounter> = meter.createCounter("requests", {
+  monotonic: true,
+  labelKeys: ["route"],
+  description: "Count all incoming requests"
+});
+
+const handles = new Map();
+
+export const countAllRequests = () => {
+  return (req, res, next) => {
+    if (!handles.has(req.path)) {
+      const labelSet = meter.labels({ route: req.path });
+      const handle = requestCount.bind(labelSet);
+      handles.set(req.path, handle);
+    }
+
+    handles.get(req.path).add(1);
+    next();
+  };
+};
+```
+
+Ensure prometheus is running by running the `prometheus` binary from earlier and start your application.
+
+```sh
+$ npm start
+
+> @opentelemetry/getting-started@1.0.0 start /Users/nina/Works/opentelemetry-js/getting-started/example/ts
```

Comment:
I would suggest to remove `/Users/...` path or replace with some fake path.

Reply:
Sure. Thank you for noticing!